tests/e2e/utils/sidebarNavigation.ts

```diff
@@ -26,9 +25,6 @@ export async function openConfluentSidebar(page: Page): Promise<void> {
     await activityBarItem.locator.click();
   }
 
-  const notificationArea = new NotificationArea(page);
-  await expect(notificationArea.errorNotifications).toHaveCount(0);
```

Comment:
Moved to the `@smoke` test since that's the only one that should test for no extension activation errors. Removing here so any callers that might deal with it after extension (see: project scaffolding tests when opening a new project window) don't fail due to potential false-positives on unrelated error notifications.